fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -325,6 +353,72 @@ class BrowserToolbarCFRPresenter(
         }
     }
 
+    @OptIn(ExperimentalComposeUiApi::class)
+    @VisibleForTesting
+    @Suppress("LongMethod")
+    internal fun showCookieBannersCFR() {
+        CFRPopup(
+            anchor = toolbar.findViewById(
+                R.id.mozac_browser_toolbar_security_indicator,
+            ),
+            properties = CFRPopupProperties(
+                popupAlignment = INDICATOR_CENTERED_IN_ANCHOR,
+                popupBodyColors = listOf(
+                    getColor(context, R.color.fx_mobile_layer_color_gradient_end),
+                    getColor(context, R.color.fx_mobile_layer_color_gradient_start),
+                ),
+                popupVerticalOffset = CFR_TO_ANCHOR_VERTICAL_PADDING.dp,
+                dismissButtonColor = getColor(context, R.color.fx_mobile_icon_color_oncolor),
+                indicatorDirection = if (settings.toolbarPosition == ToolbarPosition.TOP) {
+                    CFRPopup.IndicatorDirection.UP
+                } else {
+                    CFRPopup.IndicatorDirection.DOWN
+                },
+            ),
+            onDismiss = {
+                when (it) {
+                    true -> {
+                        TrackingProtection.tcpCfrExplicitDismissal.record(NoExtras())
+                        settings.shouldShowTotalCookieProtectionCFR = false
```

Comment:
I think here it should be settings.shouldShowCookieBannersCFR = false  ,
If it is needed for the CFR to be shown only once.

Reply:
Addressed!